mobilizon_reshare/config/config.py

```diff
@@ -61,13 +61,16 @@ def build_settings(
     with importlib.resources.path(
         mobilizon_reshare, "settings.toml"
     ) as bundled_settings_path:
-        SETTINGS_FILE = [
-            bundled_settings_path,
-            Path(dirs.site_config_dir, "mobilizon_reshare.toml"),
-            Path(dirs.user_config_dir, "mobilizon_reshare.toml"),
-            os.environ.get("MOBILIZION_RESHARE_SETTINGS_FILE"),
+        for f in [
```

Comment:
this is getting out of hand. I think we are overmanaging what DYNACONF is doing instead of just using the default behavior and just let the user specify the config file.

At this point I think this is becoming rather obscure and we should just revert to having the default behavior plus the  override with a CLI parameter that is already in place (and maybe add a CLI override for the secrets too). 

Dynaconf is designed to merge config files and we point at their docs to understand how it works. If we introduce all this stuff we would have to document a lot of custom behaviors too.